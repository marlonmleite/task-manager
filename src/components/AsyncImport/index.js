import Loadable from 'react-loadable'
import { LoadingContent } from '../Loading'

const AsyncImport = config => Loadable({
  ...config,
  loading: LoadingContent,
})

export default AsyncImport
