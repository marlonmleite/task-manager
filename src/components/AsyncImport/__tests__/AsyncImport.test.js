import React from 'react'
import { shallow } from 'enzyme'
import AsyncImport from '../'

describe('AsyncImport', () => {
  it('should render', () => {
    const comp = shallow(<AsyncImport />)

    expect(comp).toMatchSnapshot()
  })
})
