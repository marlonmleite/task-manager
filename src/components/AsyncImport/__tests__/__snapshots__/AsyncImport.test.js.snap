// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AsyncImport should render 1`] = `<Component />`;
